src/dune/assets.ml

```diff
@@ -0,0 +1,71 @@
+let jbuild_plugin_ml = {jbp|let () =
```

Comment:
Why do we have to promote this file? I guess it's because of bootstrapping?

Reply:
Yup, it's just for bootstrap